editor/src/clj/editor/cljfx_form_view.clj

```diff
@@ -963,6 +986,7 @@
                                                         9   ;; bottom scrollbar
                                                         (* line-height
                                                            (max 1 (count value))))
+                                        :column-resize-policy (custom-table-resize-policy)
```

Comment:
I tried predefined policies, but all of them turn horizontal scroll off, so they don't fit for our needs.